packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -5235,6 +5235,7 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin, Res
         child: AbsorbPointer(
           absorbing: false, // it's mutated directly by _cancelActivePointers above
           child: FocusTraversalGroup(
+            policy: FocusTraversalGroup.maybeOf(context) ?? ReadingOrderTraversalPolicy(),
```

Comment:
You probably don't even need the ?? half of that.

Reply:
Hmm. Good point!